### DISCUSSION TITLE
UX: Add a "Don't ask me again" Button on the reminder

### DISCUSSION THREAD

**Discussion by ianmuchina** [community/community#3456] [Discussions] [answered]
Having to repeatedly click the "Got it" Button every time I refresh a repo is a negative experience.

A Don't ask me again button would be nice

![Screenshot_2021-02-28_22-57-56](https://user-images.githubusercontent.com/49595512/109432730-b4269600-7a1d-11eb-87fe-574835400ddc.png)

https://user-images.githubusercontent.com/49595512/109432420-21392c00-7a1c-11eb-9b09-519c0eefbcec.mp4

**Comment by PranamBhat** [comment, marked as answer]
True. Check @evi-liu @becca
